docs/rule/attribute-indentation.md

```diff
@@ -119,3 +119,5 @@ Block form (HTML)
   * boolean - `true` - Enables the rule to be enforced when the opening invocation has more than 80 characters or when it spans multiple lines.
   * object - { 'open-invocation-max-len': n characters } - Maximum length of the opening invocation.
   * object - { 'process-elements': `true` } - Also validate the indentation of HTML/SVG elements.
+  * object - { 'end-element-open': `new-line`|`last-attribute` } - Enforce the position of the closing brace `>` to be on a new line or next to the last attribute (defaults to `new-line`).
```

Comment:
the two option names seem confusing to me, it seems like we want to be consistent. I'd prefer to call it `element-open-end`. What do you think?

Reply:
Makes sense to me. I didn't feel we had total agreement on naming so I went with the most recent suggestions and figured we'd go from there.

Should I maybe validate that the values passed are either `new-line` or `last-attribute`?
As of now, I default the values to `new-line` and throughout the code I only check if `=== 'last-attribute'` so any other value than `last-attribute` will actually behave as `new-line` which I believe could be confusing.

Reply:
> Should I maybe validate that the values passed are either new-line or last-attribute?

Yes, that makes sense, good idea.

Reply:
Done. Config values are validated now :-).